moj-be/modules/custom/moj_resources/src/SuggestedContentApiClass.php

```diff
@@ -97,16 +97,38 @@ private function translateNode(NodeInterface $node)
   private function getSuggestions($nid, $number, $prison)
   {
     $node = $this->node_storage->load($nid);
-    $detailed_results = $this->combineSecondaryTagItems($node->field_moj_secondary_tags, $number, $prison);
+    $secondary_tags = $node->field_moj_secondary_tags;
+
+    $detailed_results = $this->getBothSecondaryTagItemsFor(
```

Comment:
`Both` is a little too specific I think `getSecondaryTagItemsFor` will be fine

Reply:
Done